méli-mélo/2024-02-charts/index.html

```diff
@@ -0,0 +1,139 @@
+---
+title: Charts - Documentation
+dateModified: 2024-02-23
+description: "Create a visual representation of data in a table."
+lang: en
+script:
+- js/deps/chart.min.js
+- js/charts.js
+---
+
+<p>This component is used to generate a visual representation of data in a table.</p>
+<dl>
+	<dt>Version</dt>
+	<dd>0.1</dd>
+	<dt>Status</dt>
+	<dd>Provisional</dd>
+	<dt>Technical name</dt>
+	<dd><code>gc-chart</code></dd>
+	<dt>Type</dt>
+	<dd>Canada.ca component</dd>
+</dl>
+
+<section>
+	<h2>On this page</h2>
+	<ul>
+		<li><a href="#examples">Working examples</a></li>
+		<li><a href="#implement">How to implement</a></li>
+		<li><a href="#configuration">Configuration</a></li>
+		<li><a href="#notes">Notes</a></li>
+		<li><a href="#implementationPlan">Implementation plan</a></li>
+	</ul>
+</section>
+
+<section id="examples">
+	<h2>Working examples</h2>
+	<ul>
+		<li><a href="demo.html">Charts - Working examples</a></li>
+	</ul>
+</section>
+
+<section id="implement">
+	<h2>How to implement</h2>
+	<p>This section explains how you can implement the Charts component in your page.</p>
+	<ul>
+		<li>Add the data attribute <code>data-chart</code> to any <code>&lt;table></code> element that contains numerical data where you want a visual representation of said data. The attribute accepts the following values:
+			<ul>
+				<li><code>line</code></li>
+				<li><code>bubble</code></li>
+				<li><code>doughnut</code></li>
+				<li><code>pie</code></li>
+				<li><code>line</code></li>
+				<li><code>polarArea</code></li>
+				<li><code>radar</code></li>
+				<li><code>scatter</code></li>
```

Comment:
We will need an example of each or we can say that "only the following chart is supported : { list of chart type demoed }" and to enforce that charting limitation

This is to satisfy at demoing all feature requirement.